db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/DataDictionaryCache.java

```diff
@@ -78,7 +79,7 @@
     private ManagedCache<Long,Conglomerate> conglomerateCache;
     private ManagedCache<Pair<Long, Long>, Conglomerate> txnAwareConglomerateCache;
     private ManagedCache<Long,ConglomerateDescriptor> conglomerateDescriptorCache;
-    private ManagedCache<GenericStatement,GenericStorablePreparedStatement> statementCache;
+    private ManagedCache<StatementKey,GenericStorablePreparedStatement> statementCache;
```

Comment:
To me it makes more sense to store the Timestmap in the value rather than the key, but I don't mind.

Reply:
you're right, fixed it.